lms/static/sass/views/_homepage.scss

```diff
@@ -72,6 +72,11 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
 
       .course-info {
         height: $course-info-height;
+        font-family: $sans-serif;
```

Comment:
After updating the font family to `inherit` the course cards were using serifed type. This ensures they use the sans-serif.